Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -180,7 +181,27 @@ Please note that this issue can hold on multiple levels: When you use webviews w
 
 In order to ensure that the libraries used by the apps are not carrying vulnerabilities, one can best check the dependencies installed by CocoaPods or Carthage.
 
-In case CocoaPods is used for managing third party dependencies, the following steps can be taken to analyze the third party libraries for vulnerabilities:
+###### Swift Package Manager
+
+In case [Swift Package Manager](https://swift.org/package-manager "Swift Package Manager on Swift.org") is used for managing third party dependencies, the following steps can be taken to analyze the third party libraries for vulnerabilities:
+
+First, at the root of the project, where the Package.swift file is located, type
+
+```shell
+$ swift build
+```
+
+Next, check the Package.resolved for actual versions used and inspect the given libraries for known vulnerabilities.
+
+You can utilize the OWASP [Dependency-Check](https://owasp.org/www-project-dependency-check/ "OWASP Dependency-Check")'s experimental [Swift Package Manager Analyzer](https://jeremylong.github.io/DependencyCheck/analyzers/swift.html "dependency-check – SWIFT Package Manager Analyzer") to identify the [Common Platform Enumeration (CPE)](https://nvd.nist.gov/products/cpe) of all dependencies and any corresponding [Common Vulnerability and Exposure (CVE)](https://cve.mitre.org/) entries. Scan the application's Package.swift file and generate a report of known vulnerable libraries with the following command:
+
+```shell
+$ dependency-check  --enableExperimental --out . --scan Package.swift
+```
+
+###### CocoaPods
+
+In case [CocoaPods](https://cocoapods.org "CocoaPods.org") is used for managing third party dependencies, the following steps can be taken to analyze the third party libraries for vulnerabilities:
```

Comment:
```suggestion
In case [CocoaPods](https://cocoapods.org "CocoaPods.org") is used for managing third party dependencies, the following steps can be taken to analyze the third party libraries for vulnerabilities.
```